openc3-cosmos-init/plugins/packages/openc3-tool-base/src/main.js

```diff
@@ -2,14 +2,16 @@ import { createApp } from 'vue'
 import { defineCustomElements } from '@astrouxds/astro-web-components/loader'
 
 import App from './App.vue'
-import vuetify from './plugins/vuetify'
+import vuetify from '@openc3/tool-common/src/plugins/vuetify'
 import router from './router'
-import store from '../../packages/openc3-tool-common/src/plugins/store'
-import '../../packages/openc3-tool-common/src/assets/stylesheets/layout/layout.scss'
-import Notify from '../../packages/openc3-tool-common/src/plugins/notify'
+import store from '@openc3/tool-common/src/plugins/store'
+import '@openc3/tool-common/src/assets/stylesheets/layout/layout.scss'
+import Notify from '@openc3/tool-common/src/plugins/notify'
 
 defineCustomElements()
 
+Object.getPrototypeOf(System).firstGlobalProp = true;
```

Comment:
What does this do?

Reply:
I don't remember the technical details, but it has to do with making systemjs work with Vue 3 since there's no global `Vue` anymore like there was with Vue 2. Vue 3 is just a module import

https://github.com/systemjs/systemjs/issues/2272